Xamarin.Essentials/Calendars/Calendars.uwp.cs

```diff
@@ -9,24 +10,34 @@ namespace Xamarin.Essentials
     public static partial class Calendars
     {
         static Task<AppointmentStore> uwpAppointmentStore;
+        static AppointmentStoreAccessType lastRequestType;
 
-        static Task<AppointmentStore> GetInstanceAsync() =>
-            uwpAppointmentStore ??= AppointmentManager.RequestStoreAsync(AppointmentStoreAccessType.AllCalendarsReadOnly).AsTask();
+        static Task<AppointmentStore> GetInstanceAsync(AppointmentStoreAccessType type = AppointmentStoreAccessType.AppCalendarsReadWrite)
```

Comment:
Maybe we can have multiple fields for this? Do the stores communicate? Like if I add a new calendar, will it appear in the other store? Is this expensive? Should we even be caching this?